src/network/connection_handle.cpp

```diff
@@ -99,23 +100,23 @@ DEF_TRANSITION_GRAPH
     ON(FINISH) SET_STATE_TO(CLOSING) AND_INVOKE(CloseSocket)
     ON(PROCEED) SET_STATE_TO(PROCESS) AND_INVOKE(Process)
   END_DEF
-  
-  DEFINE_STATE(PROCESS) 
+
+  DEFINE_STATE(PROCESS)
     ON(PROCEED) SET_STATE_TO(WRITE) AND_INVOKE(ProcessWrite)
     ON(NEED_DATA) SET_STATE_TO(READ) AND_INVOKE(FillReadBuffer)
     ON(GET_RESULT) SET_STATE_TO(GET_RESULT) AND_WAIT
     ON(FINISH) SET_STATE_TO(CLOSING) AND_INVOKE(CloseSocket)
-    ON(NEED_SSL_HANDSHAKE) SET_STATE_TO(PROCESS_WRITE_SSL_HANDSHAKE) 
+    ON(NEED_SSL_HANDSHAKE) SET_STATE_TO(PROCESS_WRITE_SSL_HANDSHAKE)
       AND_INVOKE(ProcessWrite_SSLHandshake)
   END_DEF
 
-  DEFINE_STATE(WRITE) 
+  DEFINE_STATE(WRITE)
     ON(WAKEUP) SET_STATE_TO(WRITE) AND_INVOKE(ProcessWrite)
     ON(NEED_DATA) SET_STATE_TO(PROCESS) AND_INVOKE(Process)
     ON(PROCEED) SET_STATE_TO(PROCESS) AND_INVOKE(Process)
   END_DEF
 
-  DEFINE_STATE(GET_RESULT) 
+  DEFINE_STATE(GET_RESULT)
     ON(WAKEUP) SET_STATE_TO(GET_RESULT) AND_INVOKE(GetResult)
```

Comment:
These look like unintentional changes, can you revert them?

Reply:
These are ok, they can stay. They remove extra spaces at the end of lines and in otherwise blank lines.

@Nov11 for future work though, I would recommend and request that you keep functional changes separate from cleanup or cosmetic changes. Stick to just the changes. This makes it easier to review the changes.

Code cleanup is desirable, but it should be a separate PR. There is a code formatter that we'll eventually run across the code base, to clean up and standardize the style.

Reply:
Got it.  I'll stick to this rule.